lib/matplotlib/backends/backend_ps.py

```diff
@@ -1442,7 +1445,7 @@ def convert_psfrags(tmpfile, psfrags, font_preamble, custom_preamble,
     latexfile = latexfile.replace("~", "\\string~")
     command = [str("latex"), "-interaction=nonstopmode",
                '"%s"' % latexfile]
-    verbose.report(command, 'debug')
+    log.debug(command)
```

Comment:
Please use `log.debug('%s', command)` (throughout) so as not to crash if command has the bad idea of including a `%`.  See https://github.com/sphinx-doc/sphinx/issues/4070 for a similar issue (except that they mangled the fix :-/).

Reply:
OK, I can see this has an advantage.  OTOH, there is lots of this kind of formatting that was being passed to `verbose`.  I don't mind fixing the easy cases, but there are also cases like:

```python
log.info('figure size (inches) has been adjusted '
                               'from %s x %s to %s x %s' % (wo, ho, w, h))
```

I tested this, and its not as simple as:

```python 
log.info('figure size (inches) has been adjusted '
                               'from %s x %s to %s x %s', (wo, ho, w, h))
```
which fails.   

More importantly, I'm concerned that by redoing all these I'll screw something up.  These aren't the easiest parts of the code to debug since the tests don't get run with the logging turned on.  

For now, I'll fix the easy one-`%s` cases, but not the more complicated formatting.